examples/see3cam/see3cam_test.cc

```diff
@@ -95,29 +93,30 @@ int main(int argc, char *argv[])
 
         unsigned int frame = 0;
         for(frame = 0;; frame++) {
-            bool success = false;
-            switch(mode) {
+            try {
+                switch (mode) {
                 case Mode::Clamp:
-                    success = cam.captureSuperPixelClamp(output);
+                    cam.captureSuperPixelClamp(output);
                     break;
                 case Mode::Shift:
-                    success = cam.captureSuperPixel(output);
+                    cam.captureSuperPixel(output);
                     break;
                 case Mode::WhiteBalanceU30:
-                    success = cam.captureSuperPixelWBU30(output);
+                    cam.captureSuperPixelWBU30(output);
                     break;
                 case Mode::WhiteBalanceCoolWhite:
-                    success = cam.captureSuperPixelWBCoolWhite(output);
+                    cam.captureSuperPixelWBCoolWhite(output);
                     break;
                 case Mode::Greyscale:
-                    success = cam.captureSuperPixelGreyscale(output);
-            }
+                    cam.captureSuperPixelGreyscale(output);
+                }
 
-            if(success) {
                 cv::imshow("Raw", output);
 
                 unwrapper.unwrap(output, unwrapped);
                 cv::imshow("Unwrapped", unwrapped);
+            } catch (...) {
```

Comment:
Is it not possible to be more specific here?

Reply:
Hmm. Yeah it probably should only catch our ``std::runtime_error``s